accounts/models.py

```diff
@@ -82,20 +84,23 @@ class User(AbstractBaseUser):
     # for the 3 day period if they are not verified
     is_active = models.BooleanField(default=True)
     is_verified = models.BooleanField(default=False)    # used to verify user account
+    owner = models.BooleanField(default=False) # initialized to True for users who created their organization
     staff = models.BooleanField(default=False)
     admin = models.BooleanField(default=False)
     first_name = models.CharField(max_length=50)
     last_name = models.CharField(max_length=50)
     joined = models.DateTimeField(auto_now_add=True) # used in cron job to check when to delete user account
 
     terms_of_use = models.BooleanField(default=False)
-    linkedin = models.URLField(blank=True, null=True, unique=True)
+    linkedin = models.URLField(blank=True, null=True)
     photo = models.ImageField(null=True, blank=True, upload_to=get_dir)
     role = models.TextField(null=True, blank=True)
     country = models.TextField(null=True, blank=True)
```

Comment:
do you mind adding the country field here too, so we can remain consistent? its probably a better approach than my textfield thing

Reply:
yup ill modify it.